deploy/examples/k8s/base/collector.yaml

```diff
@@ -0,0 +1,105 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  namespace: holoinsight-example
+  name: collector-config
+data:
+  config.yml: |
+    receivers:
+      holoinsight_skywalking:
+        protocols:
+          grpc:
+            endpoint: 0.0.0.0:11800
+            auth:
+              authenticator: http_forwarder_auth
+        holoinsight_server:
+          http:
+            endpoint: gateway:8080
+    
+    exporters:
+      otlp:
+        endpoint: gateway:11800
+        tls:
+          insecure: true
+    
+    processors:
+      batch:
+    
+    extensions:
+      health_check:
+      pprof:
+      zpages:
+      http_forwarder_auth:
+        url: http://gateway:8080/internal/api/gateway/apikey/check
```

Comment:
Should this be 'registry.holoinsight-server' ?

Reply:
The apikey check api in gateway.